tss-esapi/src/tcti_ldr.rs

```diff
@@ -247,6 +256,16 @@ impl FromStr for TctiNameConf {
             )?));
         }
 
+        let libtpms_pattern = Regex::new(r"^libtpms(:(.*))?$").unwrap(); //should not fail
```

Comment:
You should probably add a test for this parsing as it is done beneath this function.

Reply:
done